ResearchKit/Common/ORKReviewStep.md

```diff
@@ -0,0 +1,81 @@
+<h2>Introduce</h2>
+- `ORKReviewStep`
+- `ORKReviewStepViewController`
+- `ORKReviewStepViewControllerDelegate`
+- `ORKReviewStepReviewDirection`
+
+<h2>Change</h2>
+- `ORKTaskViewController`
+
+<h2>ORKReviewStep</h2>
+<h3>properties</h3>
+    NSArray *steps
+    ORKTaskResultSource *resultSource
+    ORKReviewStepReviewDirection *reviewDirection
+<h3>use cases</h3>
+- __outside of a survey = standalone__
+  - steps array is provided upon initialization
+  - review step is at the beginning
+  - step results may not be changed
+  - review direction defaults to forward
```

Comment:
Yep, you are right. `reviewDirection` should be a property of `ORKReviewStep` instead. The discovery should be performed by the `taskViewController`